project/AppDependencies.scala

```diff
@@ -14,8 +14,8 @@ object AppDependencies {
     "uk.gov.hmrc"       %% s"domain-play-$playVersion"                        % "9.0.0",
     "uk.gov.hmrc.mongo" %% s"hmrc-mongo-play-$playVersion"                    % mongoDbVersion,
     "uk.gov.hmrc"       %% s"emailaddress-play-$playVersion"                  % "4.0.0",
-    "uk.gov.hmrc"       %% s"play-frontend-hmrc-play-$playVersion"            % "8.5.0",
-    "uk.gov.hmrc"       %% s"internal-auth-client-play-$playVersion"          % "1.10.0"
+    "uk.gov.hmrc"       %% s"play-frontend-hmrc-play-$playVersion"            % "9.6.0",
+    "uk.gov.hmrc"       %% s"internal-auth-client-play-$playVersion"          % "2.0.0"
   )
 
   val testDependencies: Seq[ModuleID] = Seq(
```

Comment:
upgrade "org.scalatest"   %% "scalatest"   to 3.2.18
upgrade "org.scalacheck"   %% "scalacheck" to 1.18.0